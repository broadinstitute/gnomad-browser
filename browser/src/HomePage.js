import queryString from 'query-string'
import React from 'react'
import styled from 'styled-components'

import { ExternalLink, List, ListItem } from '@gnomad/ui'

import DocumentTitle from './DocumentTitle'
import InfoPage from './InfoPage'
import Link from './Link'
import Searchbox from './Searchbox'
import GnomadLogo from './GnomadLogo'

const HomePage = styled(InfoPage)`
  max-width: 740px;
  margin-top: 90px;
`

const HeadingContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  margin-bottom: 1em;
`

const Heading = styled.h1`
  padding-top: 0;
  padding-bottom: 0;
  font-size: 1.2em;
  font-weight: normal;
  letter-spacing: 2px;
  text-align: center;
`

export default () => (
  <HomePage>
    <DocumentTitle />
    <HeadingContainer>
      <GnomadLogo width="60%" />
      <Heading>Genome Aggregation Database</Heading>
    </HeadingContainer>

    <Searchbox width="100%" />

    <div
      style={{
        height: '1em',
        borderBottom: '1px solid #666',
        margin: '1em 0 2em',
        textAlign: 'center',
      }}
    >
      <span
        style={{
          position: 'relative',
          top: '0.5em',
          padding: '0 0.5em',
          background: '#fafafa',
        }}
      >
        Or
      </span>
    </div>

    <List style={{ marginBottom: '2em' }}>
      <ListItem>
        <Link to="/variant-cooccurrence">Find co-occurrence of two variants</Link>
      </ListItem>
      <ListItem>
        <Link to="/downloads">Download gnomAD data</Link>
      </ListItem>
      <ListItem>
        <Link to="/publications">Read gnomAD publications</Link>
      </ListItem>
    </List>

    <p>
      Please note that gnomAD v2.1.1 and v3.1.2 have substantially different but overlapping sample
      compositions and are on different genome builds. For more information, see{' '}
      <Link to="/help/should-i-switch-to-the-latest-version-of-gnomad">
        &quot;Should I switch to the latest version of gnomAD?&quot;
      </Link>
    </p>

    <h2 style={{ fontSize: '1em' }}>Examples</h2>
    <List>
      <ListItem>
        Gene:{' '}
        <Link preserveSelectedDataset={false} to="/gene/ENSG00000169174">
          PCSK9
        </Link>
      </ListItem>
      <ListItem>
        Transcript:{' '}
        <Link preserveSelectedDataset={false} to="/transcript/ENST00000302118">
          ENST00000302118
        </Link>
      </ListItem>
      <ListItem>
        gnomAD v3.1.2 variant:{' '}
        <Link
          preserveSelectedDataset={false}
          to={{
            pathname: '/variant/1-55051215-G-GA',
            search: queryString.stringify({ dataset: 'gnomad_r3' }),
          }}
        >
          1-55051215-G-GA
        </Link>
      </ListItem>
      <ListItem>
        gnomAD v3 mitochondrial variant:{' '}
        <Link
          preserveSelectedDataset={false}
          to={{
            pathname: '/variant/M-8602-T-C',
            search: queryString.stringify({ dataset: 'gnomad_r3' }),
          }}
        >
          M-8602-T-C
        </Link>
      </ListItem>
      <ListItem>
        gnomAD v3{' '}
        <ExternalLink href="https://gnomad.broadinstitute.org/short-tandem-repeats?dataset=gnomad_r3">
          short tandem repeat
        </ExternalLink>{' '}
        locus:{' '}
        <Link
          preserveSelectedDataset={false}
          to={{
            pathname: 'short-tandem-repeat/DMD',
            search: queryString.stringify({ dataset: 'gnomad_r3' }),
          }}
        >
          DMD
        </Link>
      </ListItem>
      <ListItem>
        gnomAD v2.1.1 variant:{' '}
        <Link
          preserveSelectedDataset={false}
          to={{
            pathname: '/variant/1-55516888-G-GA',
            search: queryString.stringify({ dataset: 'gnomad_r2_1' }),
          }}
        >
          1-55516888-G-GA
        </Link>
      </ListItem>
      <ListItem>
        gnomAD v2 structural variant:{' '}
        <Link
          preserveSelectedDataset={false}
          to={{
            pathname: '/variant/DUP_2_5708',
            search: queryString.stringify({ dataset: 'gnomad_sv_r2_1' }),
          }}
        >
          DUP_2_5708
        </Link>
      </ListItem>
      <ListItem>
        Variant co-occurrence:{' '}
        <Link
          preserveSelectedDataset={false}
          to={{
            pathname: '/variant-cooccurrence',
            search: queryString.stringify({
              dataset: 'gnomad_r2_1',
              variant: ['1-55505647-G-T', '1-55523855-G-A'],
            }),
          }}
        >
          1-55505647-G-T and 1-55523855-G-A
        </Link>
      </ListItem>
    </List>

    <h2>New to gnomAD?</h2>
    <p>
      Check out these resources to learn about gnomAD and how to use it for variant interpretation.
    </p>
    <List>
      <ListItem>
        <ExternalLink href="https://arxiv.org/abs/2107.11458">
          <em>Variant interpretation using population databases: lessons from gnomAD.</em> arXiv{' '}
          2107.11458 [q-bio.GN] (2021).
        </ExternalLink>
      </ListItem>
      <ListItem>
        <ExternalLink href="https://www.broadinstitute.org/videos/mpg-primer-using-gnomad-tips-and-tricks">
          Using gnomAD - tips and tricks (video)
        </ExternalLink>
      </ListItem>
      <ListItem>
        <ExternalLink href="https://www.broadinstitute.org/videos/gnomad-using-large-genomic-data-sets-interpret-human-genetic-variation">
          gnomAD: Using large genomic data sets to interpret human genetic variation (video)
        </ExternalLink>
      </ListItem>
      <ListItem>
        <ExternalLink href="https://rarediseasegenomics.org/blog/six-lessons-for-variant-interpretation">
          Six lessons for variant interpretation
        </ExternalLink>
      </ListItem>
    </List>

    <h2>About gnomAD</h2>

    <p>
      The{' '}
      <Link preserveSelectedDataset={false} to="/about">
        Genome Aggregation Database
      </Link>{' '}
      (gnomAD) is a resource developed by an international coalition of investigators, with the goal
      of aggregating and harmonizing both exome and genome sequencing data from a wide variety of
      large-scale sequencing projects, and making summary data available for the wider scientific
      community.
    </p>
    <p>
      The v2.1.1 data set (GRCh37/hg19) provided on this website spans 125,748 exome sequences and
      15,708 whole-genome sequences from unrelated individuals sequenced as part of various
      disease-specific and population genetic studies. The v3.1.2 data set (GRCh38) spans 76,156
      genomes{' '}
      <ExternalLink href="https://gnomad.broadinstitute.org/help/what-populations-are-represented-in-the-gnomad-data">
        of diverse ancestries
      </ExternalLink>
      , selected as in v2. The gnomAD Principal Investigators and groups that have contributed data
      to the current release are listed{' '}
      <Link preserveSelectedDataset={false} to="/about">
        here
      </Link>
      .
    </p>
    <p>
      All data here are released for the benefit of the wider biomedical community, without
      restriction on use - see the{' '}
      <Link preserveSelectedDataset={false} to="/policies">
        terms of use
      </Link>
      . Sign up for our{' '}
      <ExternalLink href="https://groups.google.com/forum/#!forum/exac_data_announcements">
        mailing list
      </ExternalLink>{' '}
      for future release announcements.
    </p>
  </HomePage>
)
