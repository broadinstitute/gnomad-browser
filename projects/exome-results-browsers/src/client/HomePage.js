import React from 'react'
import styled from 'styled-components'

import { Page } from '@broad/ui'

import HomePageContent from '@browser-components/HomePageContent'

const HomePageWrapper = styled.section`
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  align-items: center;
  margin: 80px 50px 50px;
`

const HomePage = () => (
  <Page>
    <HomePageContent />
  </Page>
)

export default HomePage
