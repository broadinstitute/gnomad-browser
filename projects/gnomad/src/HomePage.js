import React from 'react'
import styled from 'styled-components'

import { ExternalLink } from '@broad/ui'

import InfoPage from './InfoPage'
import Link from './Link'
import Searchbox from './Searchbox'
import GnomadHeading from './GnomadHeading'

const HomePage = InfoPage.extend`
  display: flex;
  flex-direction: column;
  align-items: center;
  max-width: 740px;
  margin-top: 60px;
`

const HeadingContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 60%;
  margin-bottom: 10px;
`

const SubHeading = styled.h2`
  padding-top: 0;
  padding-bottom: 0;
  font-size: 1.2em;
  font-weight: lighter;
  letter-spacing: 2px;
`

export default () => (
  <HomePage>
    <HeadingContainer>
      <GnomadHeading width={'100%'} />
      <SubHeading>genome aggregation database</SubHeading>
      <Searchbox width="70%" />
      <p>
        Examples - Gene:{' '}
        <Link preserveSelectedDataset={false} to="/gene/PCSK9">
          PCSK9
        </Link>
        , Variant:{' '}
        <Link preserveSelectedDataset={false} to="/variant/1-55516888-G-GA">
          1-55516888-G-GA
        </Link>
      </p>
    </HeadingContainer>

    <p>
      The{' '}
      <Link preserveSelectedDataset={false} to="/about">
        Genome Aggregation Database
      </Link>{' '}
      (gnomAD) is a resource developed by an international coalition of investigators, with the goal
      of aggregating and harmonizing both exome and genome sequencing data from a wide variety of
      large-scale sequencing projects, and making summary data available for the wider scientific
      community.
    </p>
    <p>
      The data set provided on this website spans 125,748 exome sequences and 15,708 whole-genome
      sequences from unrelated individuals sequenced as part of various disease-specific and
      population genetic studies. The gnomAD Principal Investigators and groups that have
      contributed data to the current release are listed{' '}
      <Link preserveSelectedDataset={false} to="/about">
        here
      </Link>
      .
    </p>
    <p>
      All data here are released for the benefit of the wider biomedical community, without
      restriction on use - see the terms of use{' '}
      <Link preserveSelectedDataset={false} to="/terms">
        here
      </Link>
      . Sign up for our mailing list for future release announcements{' '}
      <ExternalLink href="https://groups.google.com/forum/#!forum/exac_data_announcements">
        here
      </ExternalLink>
      .
    </p>
  </HomePage>
)
