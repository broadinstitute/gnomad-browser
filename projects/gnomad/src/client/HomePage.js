import React from 'react'
import styled from 'styled-components'

import { ExternalLink } from '@broad/ui'

import DocumentTitle from './DocumentTitle'
import InfoPage from './InfoPage'
import Link from './Link'
import Searchbox from './Searchbox'
import GnomadHeading from './GnomadHeading'

const HomePage = styled(InfoPage)`
  display: flex;
  flex-direction: column;
  align-items: center;
  max-width: 740px;
  margin-top: 90px;
`

const HeadingContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  margin-bottom: 40px;
`

const SubHeading = styled.h2`
  padding-top: 0;
  padding-bottom: 0;
  font-size: 1.2em;
  font-weight: lighter;
  letter-spacing: 2px;
  text-align: center;
`

export default () => (
  <HomePage>
    <DocumentTitle />
    <HeadingContainer>
      <GnomadHeading width="60%" />
      <SubHeading>genome aggregation database</SubHeading>
      <Searchbox width="100%" />
      <p>
        Please note that gnomAD v2.1.1 and v3 contain largely non-overlapping samples and both
        datasets must be used to capture the full set of variation across gnomAD. For more
        information, see the FAQ{' '}
        <Link to={{ pathname: '/faq', hash: 'should-i-switch-to-the-latest-version-of-gnomad' }}>
          &quot;Should I switch to the latest version of gnomAD?&quot;
        </Link>
      </p>
      <p>
        Examples - Gene:{' '}
        <Link preserveSelectedDataset={false} to="/gene/PCSK9">
          PCSK9
        </Link>
        , Variant:{' '}
        <Link preserveSelectedDataset={false} to="/variant/1-55516888-G-GA">
          1-55516888-G-GA
        </Link>
      </p>
    </HeadingContainer>

    <p>
      The{' '}
      <Link preserveSelectedDataset={false} to="/about">
        Genome Aggregation Database
      </Link>{' '}
      (gnomAD) is a resource developed by an international coalition of investigators, with the goal
      of aggregating and harmonizing both exome and genome sequencing data from a wide variety of
      large-scale sequencing projects, and making summary data available for the wider scientific
      community.
    </p>
    <p>
      The v2 data set (GRCh37/hg19) provided on this website spans 125,748 exome sequences and
      15,708 whole-genome sequences from unrelated individuals sequenced as part of various
      disease-specific and population genetic studies. The v3 data set (GRCh38) spans 71,702
      genomes, selected as in v2. The gnomAD Principal Investigators and groups that have
      contributed data to the current release are listed{' '}
      <Link preserveSelectedDataset={false} to="/about">
        here
      </Link>
      .
    </p>
    <p>
      All data here are released for the benefit of the wider biomedical community, without
      restriction on use - see the terms of use{' '}
      <Link preserveSelectedDataset={false} to="/terms">
        here
      </Link>
      . Sign up for our mailing list for future release announcements{' '}
      <ExternalLink href="https://groups.google.com/forum/#!forum/exac_data_announcements">
        here
      </ExternalLink>
      .
    </p>
  </HomePage>
)
