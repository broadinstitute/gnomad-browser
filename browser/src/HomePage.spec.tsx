import { expect, test } from '@jest/globals'
import 'jest-styled-components'

import React from 'react'
import renderer from 'react-test-renderer'

import HomePage from './HomePage'

import { BrowserRouter } from 'react-router-dom'

test('Home Page has no unexpected changes', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <HomePage />
    </BrowserRouter>
  )
  expect(tree).toMatchSnapshot()
})
