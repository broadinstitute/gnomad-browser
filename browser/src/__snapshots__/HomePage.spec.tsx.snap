// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page has no unexpected changes 1`] = `
.c9 {
  color: #1173bb;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c9:visited,
.c9:active {
  color: #1173bb;
}

.c9:focus,
.c9:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c6 {
  padding-left: 20px;
  list-style-type: disc;
  margin-bottom: 1em;
}

.c7 {
  margin-bottom: 0.5em;
}

.c5 {
  box-sizing: border-box;
  width: 100%;
  max-width: 100%;
  height: calc(2em + 2px);
  padding: 0.375em 1.5em 0.375em 0.75em;
  border-color: #6c757d;
  border-style: solid;
  border-width: 1px;
  border-radius: 0.25em;
  background-image: url('data:image/gif;base64,R0lGODlhFQAEAIAAACMtMP///yH5BAEAAAEALAAAAAAVAAQAAAINjB+gC+jP2ptn0WskLQA7');
  background-position: center right;
  background-repeat: no-repeat;
  cursor: pointer;
  font-size: 1em;
  line-height: 1.25;
  outline: none;
  background-image: none;
}

.c5:focus {
  box-shadow: 0 0 0 0.2em rgba(66,139,202,0.5);
}

.c4 {
  height: calc(2em + 2px);
  padding: 0.375em 1.5em 0.375em 0.75em;
  border: 1px solid #6c757d;
  border-radius: 0.25em;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  background-image: url('data:image/gif;base64,R0lGODlhFQAEAIAAACMtMP///yH5BAEAAAEALAAAAAAVAAQAAAINjB+gC+jP2ptn0WskLQA7');
  background-position: center right;
  background-repeat: no-repeat;
  font-size: 1em;
  line-height: 1.25;
}

.c4:focus {
  outline: none;
  border-color: rgb(70,130,180);
  box-shadow: 0 0 0 0.2em rgba(70,130,180,0.5);
}

.c8 {
  color: #1173bb;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c8:visited,
.c8:active {
  color: #1173bb;
}

.c8:focus,
.c8:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  width: 100%;
}

.c3 select {
  border-right: 1px solid #ddd;
  border-top-right-radius: 0;
  border-bottom-right-radius: 0;
  background-color: #fff;
}

.c3 input {
  border-left: none;
  border-top-left-radius: 0;
  border-bottom-left-radius: 0;
}

.c0 {
  box-sizing: border-box;
  width: 100%;
  max-width: 1200px;
  padding: 0 15px;
  margin: 0 auto 40px;
  font-size: 14px;
  font-size: 16px;
  max-width: 740px;
  margin-top: 90px;
}

.c0 p {
  margin-bottom: 1em;
  line-height: 1.4;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  margin-bottom: 1em;
}

.c2 {
  padding-top: 0;
  padding-bottom: 0;
  font-size: 1.2em;
  font-weight: normal;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  text-align: center;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <svg
      viewBox="0 0 599.7 215.78"
      width="60%"
    >
      <title>
        gnomAD
      </title>
      <path
        d="M88.33 78.44q0 20.51-11.87 31.77t-34.4 11.26q-14.89 0-23.6-4.24V98.09a45.23 45.23 0 0 0 23 6.62q11.51 0 17.84-6.08t6.33-16.52v-5.83h-.28q-7.77 12.38-23 12.38-14.1 0-22.38-9.93T11.7 52.1q0-18.7 9.21-29.72t24.25-11q13.46 0 20.15 10.37h.28v-8.58h22.74zm-22.45-27v-5.81a18 18 0 0 0-4.13-11.91A13.38 13.38 0 0 0 51 28.79a13.93 13.93 0 0 0-11.88 5.9q-4.31 5.89-4.32 16.62 0 9.21 4.1 14.57a13.34 13.34 0 0 0 11.22 5.37 13.93 13.93 0 0 0 11.41-5.44q4.35-5.43 4.35-14.35zm113.5 35.42h-22.66V45.92q0-17.13-12.24-17.13a12.13 12.13 0 0 0-9.71 4.53A17.24 17.24 0 0 0 131 44.84v42h-22.79V13.17H131v11.66h.29q8.13-13.45 23.68-13.46 24.47 0 24.46 30.37zm53.99 1.8q-18.42 0-29-10.33t-10.54-28q0-18.28 10.93-28.61t29.58-10.33q18.35 0 28.78 10.33t10.48 27.29q0 18.34-10.76 29t-29.47 10.65zm.63-59.87q-8.07 0-12.52 5.54T217 50.02q0 21.23 17.12 21.23 16.33 0 16.34-21.81-.03-20.65-16.46-20.65zM406.69 86.86H384v-42q0-16-11.81-16.05a10.86 10.86 0 0 0-9.11 4.77 19.83 19.83 0 0 0-3.52 12v41.23h-22.74V44.4q0-15.6-11.59-15.61a11.11 11.11 0 0 0-9.31 4.6q-3.5 4.62-3.49 12.53v40.94h-22.74V13.17h22.74v11.52h.28a27.95 27.95 0 0 1 23.82-13.32q15.62 0 21.37 13.75 8.43-13.74 24.76-13.75 24 0 24 29.65zM506.75 86.96H485.4l-6.19-19.38h-30.92l-6.13 19.34h-21.22l31.65-87h23.23zm-32.02-34.38l-9.34-29.23a38.32 38.32 0 0 1-1.46-7.77h-.48a33 33 0 0 1-1.52 7.58l-9.46 29.42zm47.84 34.38v-87h30.8q46.33 0 46.33 42.39 0 20.32-12.64 32.44t-33.69 12.13zm19.58-71v55.12h9.71q12.72 0 20-7.64t7.24-20.8q0-12.42-7.18-19.55t-20.17-7.13z"
        fill="#020202"
      />
      <path
        d="M344.62 103.78H347c4.5 2.2 9.4 3.9 11.5 7.7 7.9 14.5 16.8 28.8 24.8 43.3 11 19.8 18.7 39.9 17 60.9H85.42c.2-10.5-.7-21.1.9-31.6 3.8-24.7 16.4-48.1 32.4-70.9 2.7-3.8 6.7-6.9 14.7-7.3 8.7-.4 13.4 2.9 16.4 6.8 8.5 10.9 15.3 22.2 25 32.7 5.1 5.5 12 6.1 20.2 2.1 14-6.8 18.7-16.5 24.8-25.8a72.79 72.79 0 0 1 8.5-11.1 11.94 11.94 0 0 1 9.6-2.7c6.6.9 8.4 4.6 10.7 7.8 8.1 11.2 17 22.1 28.5 32.2 4.1 3.6 10.4 5.5 18.2 5.2 7.4-.3 11.6-3.1 14.3-6.8 3.3-4.3 6.2-8.7 9.8-13 8.3-9.8 11.7-21.1 25.2-29.5zM412 215.68c2.2-21.1-1.4-42.5 6.1-63.3 4.2-11.7 15.3-22.3 27.8-32.2a45.63 45.63 0 0 1 9.3-6.2c8.5-4.1 14.6-.9 17.9 3 10.9 12.9 20.9 26 24.7 40.1 3.6 13.4 3.4 26.9 4.2 40.3.3 6.1.4 12.2.6 18.3zm101.92 0c1.4-15.8-2.2-32.1 6-47.4a351.11 351.11 0 0 1 23.8-37.6c2.6-3.6 7.6-7.1 14.9-7.7a15.9 15.9 0 0 1 14.7 6.5c8.5 10.6 20 20.7 22.5 32.3 1.8 8.3 1.6 16.7 2.3 25.1 0 .3.5.6.8.9 0 8.8-.1 17.6.1 26.4 0 1.4-.5 1.6-3.2 1.6-27.3-.2-54.6-.1-81.9-.1zm-513.6 0c-2.4-20.6 9-40.1 28.9-58.8 2.2-2 3.4-5 9.2-5.2s6.9 2.8 9.4 4.6c13.4 10 17.7 21.1 21.5 32.2 3.2 9 3.3 18.1 2.9 27.2z"
        fill="#1173bb"
      />
    </svg>
    <h1
      className="c2"
    >
      Genome Aggregation Database
    </h1>
  </div>
  <div
    className="c3"
    width="100%"
  >
    <select
      className="c4"
      onChange={[Function]}
      value="gnomad_r4"
    >
      <optgroup
        label="GRCh38"
      >
        <option
          value="gnomad_r4"
        >
          gnomAD v4.0.0
        </option>
        <option
          value="gnomad_r3"
        >
          gnomAD v3.1.2
        </option>
        <option
          value="gnomad_sv_r4"
        >
          gnomAD SVs v4
        </option>
        <option
          value="gnomad_cnv_r4"
        >
          gnomAD CNVs v4.0
        </option>
      </optgroup>
      <optgroup
        label="GRCh37"
      >
        <option
          value="gnomad_r2_1"
        >
          gnomAD v2.1.1
        </option>
        <option
          value="gnomad_sv_r2_1"
        >
          gnomAD SVs v2.1
        </option>
        <option
          value="exac"
        >
          ExAC
        </option>
      </optgroup>
    </select>
    <span
      style={
        {
          "flexGrow": 1,
        }
      }
    >
      <div
        style={
          {
            "display": "inline-block",
            "width": "100%",
          }
        }
      >
        <input
          aria-autocomplete="list"
          aria-expanded={false}
          autoComplete="off"
          className="c5"
          data-testid="searchbox-input"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Search by gene, region, or variant"
          role="combobox"
          value=""
        />
      </div>
    </span>
  </div>
  <div
    style={
      {
        "borderBottom": "1px solid #666",
        "height": "1em",
        "margin": "1em 0 2em",
        "textAlign": "center",
      }
    }
  >
    <span
      style={
        {
          "background": "#fafafa",
          "padding": "0 0.5em",
          "position": "relative",
          "top": "0.5em",
        }
      }
    >
      Or
    </span>
  </div>
  <ul
    className="c6"
    style={
      {
        "marginBottom": "2em",
      }
    }
  >
    <li
      className="c7"
    >
      <a
        className="-Link c8"
        href="/downloads"
        onClick={[Function]}
      >
        Download gnomAD data
      </a>
    </li>
    <li
      className="c7"
    >
      <a
        className="-Link c8"
        href="/publications"
        onClick={[Function]}
      >
        Read gnomAD publications
      </a>
    </li>
    <li
      className="c7"
    >
      <a
        className="-Link c8"
        href="/variant-cooccurrence"
        onClick={[Function]}
      >
        Find co-occurrence of two variants
      </a>
    </li>
  </ul>
  <p>
    Please note that the gnomAD v3 genomes are now part of gnomAD v4. For more information, see
     
    <a
      className="-Link c8"
      href="/help/should-i-switch-to-the-latest-version-of-gnomad"
      onClick={[Function]}
    >
      "Should I switch to the latest version of gnomAD?"
    </a>
  </p>
  <h2
    style={
      {
        "fontSize": "1em",
      }
    }
  >
    Examples
  </h2>
  <ul
    className="c6"
  >
    <li
      className="c7"
    >
      Gene:
       
      <a
        className="-Link c8"
        href="/gene/ENSG00000169174"
        onClick={[Function]}
      >
        PCSK9
      </a>
    </li>
    <li
      className="c7"
    >
      Transcript:
       
      <a
        className="-Link c8"
        href="/transcript/ENST00000302118"
        onClick={[Function]}
      >
        ENST00000302118
      </a>
    </li>
    <li
      className="c7"
    >
      Variant:
       
      <a
        className="-Link c8"
        href="/variant/1-55051215-G-GA?dataset=gnomad_r4"
        onClick={[Function]}
      >
        1-55051215-G-GA
      </a>
    </li>
    <li
      className="c7"
    >
      Structural variant region:
       
      <a
        className="-Link c8"
        href="/region/19-11078371-11144910?dataset=gnomad_sv_r4"
        onClick={[Function]}
      >
        19-11078371-11144910
      </a>
    </li>
    <li
      className="c7"
    >
      Copy number variant region:
       
      <a
        className="-Link c8"
        href="/region/2-49918501-51225575?dataset=gnomad_cnv_r4"
        onClick={[Function]}
      >
        19-11078371-11144910
      </a>
    </li>
    <li
      className="c7"
    >
      Mitochondrial variant:
       
      <a
        className="-Link c8"
        href="/variant/M-8602-T-C?dataset=gnomad_r4"
        onClick={[Function]}
      >
        M-8602-T-C
      </a>
    </li>
    <li
      className="c7"
    >
      <a
        className="-Link c8"
        href="/short-tandem-repeats?dataset=gnomad_r4"
        onClick={[Function]}
      >
        Short tandem repeat
         
      </a>
      locus:
       
      <a
        className="-Link c8"
        href="short-tandem-repeat/ATXN1?dataset=gnomad_r4"
        onClick={[Function]}
      >
        ATXN1
      </a>
    </li>
    <li
      className="c7"
    >
      Regional missense constraint (gnomAD v2, GRCh37):
       
      <a
        className="-Link c8"
        href="/gene/ENSG00000183454?dataset=gnomad_r2_1&variant=1-55505647-G-T&variant=1-55523855-G-A"
        onClick={[Function]}
      >
        GRIN2A
      </a>
    </li>
    <li
      className="c7"
    >
      Variant co-occurrence (gnomAD v2, GRCh37):
       
      <a
        className="-Link c8"
        href="/variant-cooccurrence?dataset=gnomad_r2_1&variant=1-55505647-G-T&variant=1-55523855-G-A"
        onClick={[Function]}
      >
        1-55505647-G-T and 1-55523855-G-A
      </a>
    </li>
  </ul>
  <h2>
    New to gnomAD?
  </h2>
  <p>
    Check out these resources to learn about gnomAD and how to use it for variant interpretation.
  </p>
  <ul
    className="c6"
  >
    <li
      className="c7"
    >
      <a
        className="c9"
        href="https://onlinelibrary.wiley.com/doi/10.1002/humu.24309"
        rel="noopener noreferrer"
        target="_blank"
      >
        <em>
          Gudmundsson et al. Variant interpretation using population databases: Lessons from gnomAD.
        </em>
         
        Hum Mutat. 2022 Aug;43(8):1012-1030.
      </a>
    </li>
    <li
      className="c7"
    >
      <a
        className="c9"
        href="https://www.broadinstitute.org/videos/mpg-primer-using-gnomad-tips-and-tricks"
        rel="noopener noreferrer"
        target="_blank"
      >
        Using gnomAD - tips and tricks (video)
      </a>
    </li>
    <li
      className="c7"
    >
      <a
        className="c9"
        href="https://www.broadinstitute.org/videos/gnomad-using-large-genomic-data-sets-interpret-human-genetic-variation"
        rel="noopener noreferrer"
        target="_blank"
      >
        gnomAD: Using large genomic data sets to interpret human genetic variation (video)
      </a>
    </li>
    <li
      className="c7"
    >
      <a
        className="c9"
        href="https://rarediseasegenomics.org/blog/six-lessons-for-variant-interpretation"
        rel="noopener noreferrer"
        target="_blank"
      >
        Six lessons for variant interpretation
      </a>
    </li>
  </ul>
  <h2>
    About gnomAD
  </h2>
  <p>
    The
     
    <a
      className="-Link c8"
      href="/about"
      onClick={[Function]}
    >
      Genome Aggregation Database
    </a>
     
    (gnomAD) is a resource developed by an international coalition of investigators, with the goal of aggregating and harmonizing both exome and genome sequencing data from a wide variety of large-scale sequencing projects, and making summary data available for the wider scientific community.
  </p>
  <p>
    The v4 data set (GRCh38) provided on this website spans 730,947 exome sequences and 76,215 whole-genome sequences from unrelated individuals, of
     
    <a
      className="-Link c8"
      href="/stats#diversity"
      onClick={[Function]}
    >
      diverse ancestries
    </a>
    , sequenced as part of various disease-specific and population genetic studies. The gnomAD Principal Investigators and team can be found 
    <a
      className="-Link c8"
      href="/team"
      onClick={[Function]}
    >
      here
    </a>
    , and the groups that have contributed data to the current release are listed
     
    <a
      className="-Link c8"
      href="/about"
      onClick={[Function]}
    >
      here
    </a>
    .
  </p>
  <p>
    All data here are released for the benefit of the wider biomedical community, without restriction on use - see the
     
    <a
      className="-Link c8"
      href="/policies"
      onClick={[Function]}
    >
      terms of use
    </a>
    . Sign up for our
     
    <a
      className="c9"
      href="https://groups.google.com/forum/#!forum/exac_data_announcements"
      rel="noopener noreferrer"
      target="_blank"
    >
      mailing list
    </a>
     
    for future release announcements.
  </p>
</div>
`;
